src/status_im2/contexts/onboarding/create_password/view.cljs

```diff
@@ -164,11 +164,12 @@
 
          [rn/view {:style style/bottom-part}
           [rn/view {:style style/disclaimer-container}
-           [quo/disclaimer
-            {:blur?     true
-             :on-change #(reset! accepts-disclaimer? %)
-             :checked?  @accepts-disclaimer?}
-            (i18n/label :t/password-creation-disclaimer)]]
+           [rn/touchable-opacity {:on-press #(swap! accepts-disclaimer? not)}
+            [quo/disclaimer
+             {:blur?     true
+              :on-change #(reset! accepts-disclaimer? %)
```

Comment:
Is this needed now?

Reply:
yes, otherwise tap on the checkbox doesn't work 😅

Reply:
I mean an exact tap on the checkbox square area won't do anything, otherwise clicking on the row will work

Reply:
I was thinking of modifying the `[quo/disclaimer` component to incorporate this behaviour, but I am not sure of its impact on other places.

Reply:
it's probably better to just update at quo component because I'm sure it is a common issue across other screens where it's used

Reply:
@J-Son89  : addressed here https://github.com/status-im/status-mobile/pull/15757/commits/b7cca93a85c48b38994901dc80509f4d322d7e26